var/spack/repos/builtin/packages/python/package.py

```diff
@@ -1272,8 +1272,9 @@ def setup_dependent_build_environment(
         """
         # The logic below is linux specific, and used to inject the compiler wrapper to
         # compile Python extensions. Thus, it is not needed on Windows.
-        if sys.platform == "win32":
-            return
+        if self.spec.satisfies("platform=windows"):
+            env.prepend_path("PATH", dependent_spec.prefix.scripts)
+            env.prepend_path("PATH", self.spec.prefix)
```

Comment:
Based on above comment, this should still return at the end of the block (so either a return should be added or the comment should change).

Reply:
Good catch